### PR TITLE
-Modified supplier class and fiche to accept start and end dates in lines

### PR DESCRIPTION
**Pull request by IonAgorria**
This patch enables fetching and updating of start_date and end_date columns in commande_fournisseurdet table

### CODE REVIEW

File: htdocs/fourn/commande/fiche.php [hunk 1]
```diff
@@ -182,6 +185,8 @@
 	// Set if we used free entry or predefined product
 	$predef='';
 	$product_desc=(GETPOST('dp_desc')?GETPOST('dp_desc'):'');
+	$date_start=dol_mktime(GETPOST('date_start'.$date_pf.'hour'), GETPOST('date_start'.$date_pf.'min'), 0, GETPOST('date_start'.$date_pf.'month'), GETPOST('date_start'.$date_pf.'day'), GETPOST('date_start'.$date_pf.'year'));
+	$date_end=dol_mktime(GETPOST('date_start'.$date_pf.'hour'), GETPOST('date_start'.$date_pf.'min'), 0, GETPOST('date_end'.$date_pf.'month'), GETPOST('date_end'.$date_pf.'day'), GETPOST('date_end'.$date_pf.'year'));
```
Comment (aspangaro) [inline htdocs/fourn/commande/fiche.php:189]: date_start into the line of date_end, I think is a mistake.

**Comment by aspangaro** [comment]
The version 3.6 of Dolibarr is freeze, no new function is authorized. 

You can closed this PR and proposed into develop branch

**Comment by IonAgorria** [comment]
You are right, there is a mistake in date_start, I will fix that

On the other side, this can be considered a bugfix because on 3.6 when you create a supplier order, appears a date selector but is not active, the reason of this pull request is to make it usable

**Comment by IonAgorria** [comment]
I messed up... I am going to make a correct pull request at develop branch

**Pull request closed without merging**